### PR TITLE
add mfa support

### PR DESCRIPTION
**Pull request by rinzler99**
Task: https://toruslabs.atlassian.net/browse/PD-1019

Demo:https://www.loom.com/share/d9fc9663f53b40628d852217da3eb812

### CODE REVIEW

File: src/store/mutations.js [hunk 1]
```diff
@@ -156,6 +156,12 @@ export default {
       skipTKey: payload || false,
     }
   },
+  setMfaLevel(state, payload) {
+    state.embedState = {
+      ...state.embedState,
+      mfaLevel: payload || 'optional',
```
Comment (himanshuchawla009) [inline src/store/mutations.js:162]: not neeed of  || 'optional' here

File: src/store/setupStoreChannels.js [hunk 2]
```diff
@@ -69,6 +69,7 @@ if (!isMain) {
         torusWidgetVisibility = true,
         loginConfig = {},
         skipTKey = false,
+        mfaLevel = 'optional',
```
Comment (himanshuchawla009) [inline src/store/setupStoreChannels.js:72]: keep it null by default

File: src/store/state.js [hunk 3]
```diff
@@ -63,6 +63,7 @@ const initialState = {
     torusWidgetVisibility: true,
     apiKey: 'torus-default',
     skipTKey: false,
+    mfaLevel: 'optional',
```
Comment (himanshuchawla009) [inline src/store/state.js:66]: keep it null

**Review by himanshuchawla009 — CHANGES_REQUESTED** [review]

File: src/store/mutations.js (hunk 1 above)
Comment (rinzler99) [reply src/store/mutations.js]: resolved

File: src/store/state.js [hunk 4]
```diff
@@ -63,6 +63,7 @@ const initialState = {
     torusWidgetVisibility: true,
     apiKey: 'torus-default',
     skipTKey: false,
+    mfaLevel: null,
```
Comment (chaitanyapotti) [inline src/store/state.js:66]: use 'default' as default value

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**